src/Symfony/Bundle/FrameworkBundle/Command/ContainerLintCommand.php

```diff
@@ -47,15 +51,32 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $container = $this->getContainerBuilder();
+        $io = new SymfonyStyle($input, $output);
+        $errorIo = $io->getErrorStyle();
+
+        try {
+            $container = $this->getContainerBuilder();
+        } catch (RuntimeException $e) {
+            $errorIo->error($e->getMessage());
+
+            return 2;
+        }
 
         $container->setParameter('container.build_hash', 'lint_container');
         $container->setParameter('container.build_time', time());
         $container->setParameter('container.build_id', 'lint_container');
 
         $container->addCompilerPass(new CheckTypeDeclarationsPass(true), PassConfig::TYPE_AFTER_REMOVING, -100);
 
-        $container->compile();
+        try {
+            $container->compile();
+        } catch (InvalidArgumentException $e) {
+            $errorIo->error($e->getMessage());
+
+            return 1;
+        }
+
+        $errorIo->success('The injected values in the services seem to be valid.');
```

Comment:
would this be a bit better?
`The container was lint successfully: all services are injected with values that are compatible with their type declarations.`